### ISSUE TITLE
EIZO3895

### ISSUE THREAD

**Issue by christianmeier** [MonitorControl/MonitorControl#351]
Hi, I can't adjust the brightness and contrast via the app. 
BigSur
11.1 (20C69)
Monitor: Eizo EV3895

Is this monitor supported / how can I troubleshoot or modify the settings to get the app working.
BR C

**Comment by JoniVR** [comment]
As the issue template (which you seem to have removed?) notes, check out the [Monitor Troubleshooting](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting) wiki.

**Comment by christianmeier** [comment]
Hi Joni, thank you for getting back to me - Sorry I did not find the troubleshooting template. 
Installed the the tool and it also does not change  any parameter.

christianmeier@iMac  ~/scripts/ddcctl  master  ./ddcctl -d 1 -b 40 -w 20000                                               ✔  1187  09:56:34
D: NSScreen #1892440164 (3840x1600 0°) 111.00 DPI
I: found 1 external display
I: polling display 1's EDID
I: got edid.serial: 20776100
I: got edid.name: EV3895
D: action: b: 40
D: setting VCP control #16 => 40
E: Failed to send DDC command!

—> Failed to send| I use the DisplayPort on my AMD RX 580 gpu / I run a Hackintosh. 

I write an email to Eizo and ask them if they changed the specification. May this type of monitor will not work because the manual says that the contract
and brightness will automatically adjusted.

Thank you for you help - If I get any feedback from EIZO I will post it.

Best regards,
Christian

**Comment by gvstavnev** [comment]
I have the EV2785 and because of the same reason I recently had contact with the german support.

`wir verwenden seit einigen Jahren nur noch Controller ohne DDC/CI. Steuerungsaufgaben sind nur über USB und mit unserer Software Screen InStyle möglich. Ich gehe davon aus, dass die anderen Hersteller die Kommunikation mit deren Software ebenfalls über USB vornehemen. Das /CI hat also kaum noch eine Bedeutung.
DDC, also die einseitige Kommunikation wird natürlich weiterhin verwendet. Alle Monitore sind also Plug-and-Play. Sie können über das Signalkabel aber keine Steurungsfunktionen übertragen.`

English:
`we have been using only controllers without DDC/CI for some years now. Control tasks are only possible via USB and with our Screen InStyle software. I assume that the other manufacturers also communicate with their software via USB. So the /CI has hardly any meaning anymore.
DDC, the one-way communication, is of course still used. So all monitors are plug-and-play. However, they cannot transmit any control functions via the signal cable.'

tl;dr
`We dont care – use our windows software.`


I think it may be possible to reverse engineer the usb protocoll. It may be nothing different than DDC.
I tried using [ddcctl](https://github.com/kfix/ddcctl) via usb-c but had no luck.

**Comment by waydabber** [comment]
We'd like to solve this issue here:

https://github.com/MonitorControl/MonitorControl/issues/364

I'll close this thread so we don't have a duplicate issue on this. Please feel free to follow the other issue and contribute with your commands or when the time comes by testing!

Also - what I would like to know if these USB MCCS EIZO displays are controlled by the macOS built in brightness slider or not (as they should be using the same protocol as Apple's displays)? Please let me know in the other thread! Thank you!
